### ISSUE TITLE
Convergence speed of Rank model

### ISSUE THREAD

**Issue by luofan18** [xialeiliu/RankIQA#51]
Hi, 

I implemented RankIQA model in tensorflow with Inception-v3 as backbone. I found that the rank netwok coverged very faster on synthetic pairs. Basically, the training loss decreased to 0 after hunderds of steps, even before 1st epoch ended. It that the same case for orignal implementation in caffe?

**Comment by xialeiliu** [comment]
I don't think that's the case. When you sample the patches, make sure to sample it at different positions of each image.

**Comment by luofan18** [comment]
I see, I'll try to use crops instead of whole picture.
